server/plugin.go

```diff
@@ -85,12 +85,11 @@ func (p *Plugin) OnActivate() error {
 	scrapeInterval := *p.configuration.ScrapeIntervalSeconds
 
 	// TODO(isacikgoz): Use multiple targets for HA env
-	ls := labels.FromMap(map[string]string{
-		model.AddressLabel:        "localhost:8067",
+	lb := labels.NewBuilder(labels.FromMap(map[string]string{
+		model.AddressLabel:        *appCfg.ServiceSettings.SiteURL,
```

Comment:
Is this right? It should be `MetricsSettings.ListenAddress` right?

Reply:
Ah right, I mixed it up.